unison-cli/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1401,6 +1392,16 @@ notifyUser dir o = case o of
           "",
           "Did you mean to use " <> IP.makeExample' IP.pushCreate <> " instead?"
         ]
+  GistCreated hqLength repo hash ->
+    pure $
+      P.lines
+        [ "Gist created. Pull via:",
+          "",
+          P.indentN 2 (IP.patternName IP.pull <> " " <> prettyRemoteNamespace remoteNamespace)
```

Comment:
I don't see any changes to the pull commands, do they already support pulling a root namespace hash? I'd love to see a test for that to make sure it keeps working in the future 😄

Reply:
They already "support" pulling a namespace hash, it doesn't look like there are any existing tests for it though. 😬 

But it would be add a test that uses `gist` to push and then the existing pull to pull.  

`Unison.Test.GitSync.pushPullTest` is a good example for how to create a git repo and run some push pull commands on it, even if you don't end up calling that function directly.

Reply:
I've pushed a test that pushes a gist, then pulls it: https://github.com/unisonweb/unison/pull/2657/commits/feef9dbd5a665a73416f51af6642b79d58be9303